### PR TITLE
[MODFIN-358] - Add tenantId to the locations schema for restrict by F…

### PR DESCRIPTION
**Pull request by imerabishvili**
## Purpose
https://folio-org.atlassian.net/browse/MODFIN-358

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Examples exist for all schemas
  - [ ] Descriptions exist for all schema properties
  - [ ] All schemas pass raml-cop linting
- Does this introduce breaking changes?
  - [ ] Were there any schema changes?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: mod-finance/examples/fund.sample [hunk 1]
```diff
@@ -23,9 +23,13 @@
     "077274ad-6b4f-4c28-9779-6d381e7a1ca1"
   ],
   "restrictByLocations": true,
-  "locationIds": [
-    "eb2d063a-5b4c-4cab-8db1-5fc5c5941df6",
-    "fcd64ce1-6995-48f0-840e-89ffa2288371"
+  "locations": [
+    {
+      locationId: "eb2d063a-5b4c-4cab-8db1-5fc5c5941df6"
```
Comment (SerhiiNosko) [inline mod-finance/examples/fund.sample:28]: thinking do we need to add tenantId to location always or only when ECS mode is enabled, if yes - we need to add tenantId to samples also, but we can remain as is and back to this question again during implementation

File: mod-finance/examples/fund.sample [hunk 2]
```diff
@@ -23,9 +23,13 @@
...
+    },
+    {
+      locationId: : "fcd64ce1-6995-48f0-840e-89ffa2288371"
```
Comment (SerhiiNosko) [inline mod-finance/examples/fund.sample:31]: two occurences of :
And the same also in other places

Comment (imerabishvili) [reply mod-finance/examples/fund.sample]: removed

File: mod-finance/examples/fund_collection.sample [hunk 3]
```diff
@@ -25,9 +25,13 @@
         "077274ad-6b4f-4c28-9779-6d381e7a1ca1"
       ],
       "restrictByLocations": true,
-      "locationIds": [
-        "eb2d063a-5b4c-4cab-8db1-5fc5c5941df6",
-        "fcd64ce1-6995-48f0-840e-89ffa2288371"
+      "locations": [
+        {
+          locationId: "eb2d063a-5b4c-4cab-8db1-5fc5c5941df6"
```
Comment (SerhiiNosko) [inline mod-finance/examples/fund_collection.sample:30]: per json syntax keys also should be wrapped with ""

Comment (imerabishvili) [reply mod-finance/examples/fund_collection.sample]: thanks, also fixed extra comma

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**